libs/prebuilt/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -252,8 +252,8 @@ def create_react_agent(
     model: Union[
         str,
         LanguageModelLike,
-        Callable[[StateSchema, Runtime[ContextT]], BaseChatModel],
-        Callable[[StateSchema, Runtime[ContextT]], Awaitable[BaseChatModel]],
+        Callable[[StateSchema, Runtime[ContextT]], LanguageModelLike],
```

Comment:
NO! I requested to widen the accepted types.

It should be BaseChatModel or Runnable[LanguageModelInput, BaseMessage].